### PR TITLE
Display alert on "candidate" activity posts with prompt to "adapt" them, fixes #839

### LINKED ISSUE #839: Display alert on "candidate" activity posts with prompt to "adapt" them
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled.

****

Our new [activities system](https://publiclab.org/tag/activity-grids) sets out a [more structured format](https://publiclab.org/notes/warren/09-17-2016/what-makes-a-good-activity) for activity posts, but we've been seeding it with existing content that's _close to_ an activity, or from which an activity could be adapted. 

For example, many activities listed [on the spectrometry page](https://publiclab.org/wiki/spectrometry#Activities) are not really formatted as activities yet, but we hope folks will adapt their content and post new ones, as [this](https://publiclab.org/notes/warren/09-18-2016/see-the-fraunhofer-absorption-lines-in-sunlight) was adapted from [this earlier post](https://publiclab.org/notes/cfastie/3-2-2013/fraunhofer).

To achieve this, we could use a [power tag](https://publiclab.org/wiki/power-tags) such as `status:candidate` -- that way it'd be listed on an Activities grid, but there'd be an acknowledgement that it's not really an activity yet. The tag could trigger an alert above the text, similarly to this code:

https://github.com/publiclab/plots2/blob/master/app/views/notes/show.html.erb#L4

The note could say something like (input, @steviepubliclab or @ebarry or @gretchengehrke?): 

> This was marked as a candidate activity -- read about how to adapt its content into a fully-fledged, step-by-step activity that can be easily replicated.

### PR DESCRIPTION
**Pull request by 500swapnil**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request is descriptively named with #number reference back to original issue
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [ ] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/views/notes/show.html.erb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 <div class="col-md-9 note-show<% if @node.status == 4 || @node.status == 0 %> moderated<% end %>">
   <% if current_user && @node.tags.length == 0 %><div class="alert alert-warning"><%= raw t('notes.show.note_no_tags') %></div><% end %>
+  <% if @node.has_power_tag('status:candidate') %><div class="alert alert-info"><a href="https://publiclab.org/notes/warren/09-17-2016/what-makes-a-good-activity" >read about how to adapt its content</a></div>
```
Comment (jywarren) [inline app/views/notes/show.html.erb]: Ah - actually we should have this whole line -- sorry i wasn't specific enough and thank you!

> This was marked as a candidate activity -- [read about how to adapt its content]() into a fully-fledged, step-by-step activity that can be easily replicated.

**Comment by jywarren** [comment]
Also note how I added `fixes #839` -- having that word in the title or commit message will auto-close the issue when it's merged -- magical! 🎩 🐇

**Comment by 500swapnil** [comment]
Okay I will add it. Thanks!

**Comment by jywarren** [comment]
Super! Thanks!

**Comment by 500swapnil** [comment]
But why was it failing tests?

**Comment by jywarren** [comment]
Oh, i'm not sure but manually checking it showed that it's OK -- you know, actually I think merging too fast can cause it to fail since it may no longer be able to access the code from the pull request. 

I guess i should always let them run through before merging.

Tests will run on master automatically though so it's OK!

Curious - do you think this could be an interesting next one to try? https://github.com/publiclab/plots2/issues/397

**Comment by jywarren** [comment]
It's multi-part, and a bit more complex, but I think you've gotten the idea of power tags well, and are ready for something like that -- and I'm happy to help!

**Comment by 500swapnil** [comment]
I can have a look at it. Thanks!

**Pull request merged**